src/ln/channel.rs

```diff
@@ -2000,6 +2000,16 @@ impl Channel {
 		self.channel_value_satoshis
 	}
 
+	#[cfg(test)]
+	pub fn get_local_keys(&self) -> &ChannelKeys {
```

Comment:
Well, I found this utilities useful if we want to test messages field by field without exposing field in API but maybe we don't want a proliferation of these ones..

Reply:
I think its fine for test-only for now, the need should go away once we have a more robust solution for #81 (ie where users get to specify how to generate the keys, meaning a ChannelManager-driver can easily re-derive the same keys). Maybe leave a TODO to that effect?